near/src/config.rs

```diff
@@ -417,11 +417,16 @@ pub struct AccountInfo {
     pub amount: Balance,
 }
 
+pub const CONFIG_VERSION: u32 = 1;
+
 /// Runtime configuration, defining genesis block.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct GenesisConfig {
     /// Protocol version that this genesis works with.
     pub protocol_version: u32,
+    /// This is a version of a genesis config structure.
+    /// It's not a major protocol version, but used for automatic config migrations using scripts.
+    pub config_version: u32,
```

Comment:
I wonder whether it should precede the protocol version.

Reply:
updated